lib/Format/UnwrappedLineFormatter.cpp

```diff
@@ -853,7 +853,14 @@ UnwrappedLineFormatter::format(const SmallVectorImpl<AnnotatedLine *> &Lines,
          TheLine.First->TokenText == "qWarning") &&
          TheLine.Last && TheLine.Last->is(tok::semi))
       ShouldFormat = false;
-    // DVLT SPECIFIC
+
```

Comment:
No, because you still can use q and t logger without << such as qDebug( "am i wrong?" )